### PR TITLE
Add support for BF SDE 9.3.0

### PR DESCRIPTION
**Pull request by bocon13**
This PR also makes 9.3.0 the default SDE for building Stratum

TODO
- [x] ports don't come up
- [x] test on hardware
- [x] run ptf tests
- [x] verify 9.2 patches are in 9.3 (for @Yi-Tseng)

### CODE REVIEW

File: bazel/deps.bzl [hunk 1]
```diff
@@ -24,6 +24,7 @@ BF_SDE_PI_VER = {
     "9_0_0": "ca0291420b5b47fa2596a00877d1713aab61dc7a",
     "9_1_0": "41358da0ff32c94fa13179b9cee0ab597c9ccbcc",
     "9_2_0": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
+    "9_3_0": "4546038f5770e84dc0d2bba90f1ee7811c9955df",
```
Comment (bocon13) [inline bazel/deps.bzl:27]: It is the same:

```
COMMAND: [git clone https://github.com/p4lang/PI.git]
COMMAND: [git checkout 4546038f5770e84dc0d2bba90f1ee7811c9955df]
```

File: stratum/hal/bin/barefoot/README.build.md [hunk 2]
```diff
@@ -20,8 +20,9 @@ access P4 Studio SDE. Contact Intel for more details.*
 #### Supported SDE versions
 
  - 9.0.0 (Deprecated)
- - 9.1.0 (Deprecated)
- - 9.2.0 (LTS release)
+ - 9.1.0 (LTS release)
+ - 9.2.0 (Deprecated)
```
Comment (bocon13) [inline stratum/hal/bin/barefoot/README.build.md:24]: confirm this

Comment (bocon13) [reply stratum/hal/bin/barefoot/README.build.md:24]: 9.1 and 9.3 are LTS releases

Comment (bocon13) [reply stratum/hal/bin/barefoot/README.build.md:24]: Clarify depreciation policy (current and LTS)

File: bazel/deps.bzl [hunk 3]
```diff
@@ -24,6 +24,7 @@ BF_SDE_PI_VER = {
     "9_0_0": "ca0291420b5b47fa2596a00877d1713aab61dc7a",
```
Comment (bocon13) [inline bazel/deps.bzl:24]: ^^ drop these

Comment (bocon13) [reply bazel/deps.bzl:24]: do this after the 2020-12 release

File: stratum/hal/bin/barefoot/README.build.md [hunk 4]
```diff
@@ -19,9 +19,10 @@ access P4 Studio SDE. Contact Intel for more details.*
 
 #### Supported SDE versions
 
- - 9.0.0 (Deprecated)
- - 9.1.0 (Deprecated)
- - 9.2.0 (LTS release)
+ - 9.0.0 (Deprecated; to be removed after 2020-12 release)
```
Comment (bocon13) [inline stratum/hal/bin/barefoot/README.build.md:22]: Per our "policy", we could actually remove this now. Any thoughts?

**Review by pudelkoM — DISMISSED** [review]
Built the 9.3.0 SDE from scratch with the packaged script and Stratum-bf inside the Builder container. Starts without issues. Have not run data plane tests.

File: stratum/hal/bin/barefoot/README.build.md (hunk 4 above)
Comment (bocon13) [reply stratum/hal/bin/barefoot/README.build.md:22]: Will remove in a follow up PR

Comment (bocon13) [reply stratum/hal/bin/barefoot/README.build.md:22]: https://github.com/stratum/stratum/pull/431

**Comment by pudelkoM** [comment]
I also completed a full run of the Fabric TNA PTF suite:

```
./ptf/run/tm/run fabric-spgw-int
...
OK
************************************************
ALL PTF TESTS PASSED :)
************************************************
```

Interestingly, the 9.3.0 driver can work with 9.2.0 compiler output and model.
Only change necessary was removing port "0/0" from the chassis config. It failed the port id lookup we do now.

ENV:
```
PWD=/Users/max/projects/fabric-tna
SDE_DOCKER_IMG=opennetworking/bf-sde:9.2.0
STRATUM_BF_DOCKER_FLAG=-v /Users/max/projects/fabric-tna/stratum_bfrt:/usr/bin/stratum_bfrt
STRATUM_BF_DOCKER_IMG=registry.aetherproject.org/tost/stratum-bfrt:latest-9.3.0
```

**Comment by bocon13** [comment]
Great, thanks @pudelkoM !

**Pull request merged**